content/en/tracing/trace_collection/compatibility/java.md

```diff
@@ -234,21 +234,22 @@ Don't see your desired datastores? Datadog is continually adding additional supp
 
 `dd-java-agent` includes support for automatically tracing the following frameworks.
 
-| Framework         | Versions   | Support Type    | Instrumentation Names (used for configuration) |
-|-------------------|------------|-----------------|------------------------------------------------|
-| Datanucleus JDO   | 4.0+       | Fully Supported | `datanucleus`                                  |
-| Dropwizard Views  | 0.7+       | Fully Supported | `dropwizard`, `dropwizard-view`                |
-| GraphQL           | 14.0+      | Fully Supported | `graphql-java`                                 |
-| Hibernate         | 3.5+       | Fully Supported | `hibernate`, `hibernate-core`                  |
-| Hystrix           | 1.4+       | Fully Supported | `hystrix`                                      |
-| JSP Rendering     | 2.3+       | Fully Supported | `jsp`, `jsp-render`, `jsp-compile`             |
-| JUnit             | 4.1+, 5.3+ | Fully Supported | `junit`, `junit-4`, `junit-5`                  |
-| Project Reactor   | 3.1+       | Fully Supported | `reactor-core`                                 |
-| Quartz            | 2.x        | Fully Supported | `quartz`                                       |
-| RxJava            | 2.x        | Fully Supported | `rxjava`                                       |
-| Spring Data       | 1.8+       | Fully Supported | `spring-data`                                  |
-| Spring Scheduling | 3.1+       | Fully Supported | `spring-scheduling`                            |
-| Twilio SDK        | < 8.0      | Fully Supported | `twilio-sdk`                                   |
+| Framework         | Versions   | Support Type                                                     | Instrumentation Names (used for configuration) |
+|-------------------|------------|------------------------------------------------------------------|------------------------------------------------|
+| Datanucleus JDO   | 4.0+       | Fully Supported                                                  | `datanucleus`                                  |
+| Dropwizard Views  | 0.7+       | Fully Supported                                                  | `dropwizard`, `dropwizard-view`                |
+| GraphQL           | 14.0+      | Fully Supported                                                  | `graphql-java`                                 |
+| Hazelcast         | 3.6+       | [Beta](#framework-integrations-disabled-by-default)              | `hazelcast`, `hazelcast_legacy`                |
```

Comment:
I think they don't put link ref into the table but later in the document to keep it short.

Reply:
The other tables seem also to have those links for beta but it's true that I forgot to add instructions on how to enable this one on the beta table. I'm adding it thanks